pkg/restore/import.go

```diff
@@ -328,6 +357,55 @@ func (importer *FileImporter) downloadSST(
 	return &sstMeta, nil
 }
 
+func (importer *FileImporter) downloadRawKVSST(
+	regionInfo *RegionInfo,
+	file *backup.File,
+) (*import_sstpb.SSTMeta, error) {
+	id, err := uuid.New().MarshalBinary()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	// Empty rule
+	var rule import_sstpb.RewriteRule
+	sstMeta := getSSTMetaFromFile(id, file, regionInfo.Region, &rule)
+
+	// Cut the SST file's range to fit in the restoring range.
+	if bytes.Compare(importer.rawStartKey, sstMeta.Range.GetStart()) > 0 {
+		sstMeta.Range.Start = importer.rawStartKey
+	}
+	// TODO: importer.RawEndKey is exclusive but sstMeta.Range.End is inclusive. How to exclude importer.RawEndKey?
```

Comment:
I saw the only use of sstMeta in tikv is before [ingest](https://github.com/tikv/tikv/blob/113b3987d63885f602857be7ad6ad906a8811988/components/raftstore/src/store/fsm/apply.rs#L2153),  this TODO seems already fixed by tikv `download` already. @kennytm  PTAL

Reply:
The Download() in TiKV's sst_importer is expecting an inclusive `sstMeta.Range.End`, and returned range is also inclusive. What do you mean by "fixed"?

Reply:
I mean `exclude importer.RawEndKey` has been done already during `download`. because`RawEndKey` has not been backup.